### PR TITLE
OZ-726: Add ability to provide extra Service metadata

### PR DESCRIPTION
**Pull request by enyachoke**
This PR introduces the ability to set extra service [meta](https://developer.hashicorp.com/consul/api-docs/catalog#meta). A good example is to provide a custom metric_path for Prometheus  when the metrics are exposed in a non standard path. 

The PR also removed the coverage folder which should not be checked in.

Ticket --> https://mekomsolutions.atlassian.net/browse/OZ-726

### CODE REVIEW

File: coverage/coverage.json
Comment (rbuisson) [inline coverage/coverage.json:1]: Should this be gitignored?

Comment (enyachoke) [reply coverage/coverage.json:1]: It should.

Comment (enyachoke) [reply coverage/coverage.json:1]: The folder is generated every time we run npm test.

Comment (rbuisson) [reply coverage/coverage.json:1]: Oh, so it is already gitignored and the coverage.json file was just a left over. Got it.

**Review by rbuisson — APPROVED** [review]

**Pull request merged**